MapboxNavigation/StyleManager.swift

```diff
@@ -153,6 +155,7 @@ open class StyleManager: NSObject {
                 currentStyle = style
                 style.apply()
                 delegate?.styleManager?(self, didApply: style)
+                postDidApplyStyleNotification(style: style)
```

Comment:
These four lines are repeated in a few places; it would be more maintainable to set `currentStyle`, apply the style, and call the delegate method within the new method.